doc/testssl.1.md

```diff
@@ -383,13 +386,56 @@ Except the environment variables mentioned above which can replace command line
 * MAX_OSSL_FAIL: A number which tells testssl.sh how often an OpenSSL s_client connect may fail before the program gives up and terminates. The default is 2. You can increase it to a higher value if you frequently see a message like *Fatal error: repeated TCP connect problems, giving up*.
 * MAX_HEADER_FAIL: A number which tells testssl.sh how often a HTTP GET request over OpenSSL may return an empty file before the program gives up and terminates. The default is 3. Also here you can incerase the threshold when you spot messages like *Fatal error: repeated HTTP header connect problems, doesn't make sense to continue*.
 
+### GRADING
+This script has a near-complete implementation of SSLLabs's '[SSL Server Rating Guide](https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide)'.
+
+This is *not* a reimplementation of the [SSLLab's SSL Server Test](https://www.ssllabs.com/ssltest/analyze.html), but a implementation of the above grading specification, slight discrepancies might occur!
+
+Disclaimer: Having a good grade does **NOT** necessary equal to having good security! Never rely solely on a good grade!
+
+As of writing, these checks are missing:
+* Authenticated encryption (AEAD) - should be graded **B** if not supported
```

Comment:
That could also be retrieved, right?

Reply:
I think it could yes. I couldn't really figure out where it would be sensible, without introducing big changes in the code.
You could get all ciphers that support AEAD, and see if any of them is offered. But I feel like there is a better way